client/src/components/pages/Stats.js

```diff
@@ -132,6 +138,95 @@ export default function Stats({ tourney, user }) {
     message.success("Updated stats visibility");
   };
 
+  const toggleEditMode = async () => {
+    const stageStatsCopy = JSON.parse(JSON.stringify(state.stageStats));
+    setState({ ...state, inEditMode: !state.inEditMode, stageStatsEdit: stageStatsCopy });
+  };
+
+  const submitEditedStageStats = async () => {
+    const updatedStats = await post("/api/stage-stats", { stats: state.stageStatsEdit });
+    fetchData();
```

Comment:
can this just use `updateStats` instead of re-fetching all the data again?
(see related comment below)

Reply:
sure... i just have to change a bunch of stuff :maskeh: